modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To access the Thanos Querier API endpoints.
+* To manage alert silences.
+* To monitor the Alertmanager API.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
+
+|===
+|Name |Description |Project
+
+|`cluster-monitoring-metrics-api` 
+|Users with this role have the ability to access Thanos Querier API endpoints. Additionally, it grants access to user-defined Prometheus and Thanos Ruler API endpoints. 
```

Comment:
there's no way to access the user-defined Prometheus API.
It also provides access to the Administrator metrics and alerts pages.
```suggestion
|Users with this role have the ability to access Thanos Querier API endpoints. Additionally, it grants access to the core platform Prometheus API and user-defined Thanos Ruler API endpoints. 
```